From 2a8ed66e94cd869a56591c3e5fcc503c913b89b7 Mon Sep 17 00:00:00 2001
From: mintsuki <mintsuki@protonmail.com>
Date: Mon, 27 Dec 2021 11:29:01 +0100
Subject: [PATCH] Lyre specific changes

---
 configure | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/configure b/configure
index 421cf85..30c35c4 100755
--- a/configure
+++ b/configure
@@ -6423,6 +6423,10 @@ CF_EOF
 		LINK_PROGS="$SHELL ${rel_builddir}/mk_prog.sh"
 		LINK_TESTS="$SHELL ${rel_builddir}/mk_prog.sh"
 		;;
+	(lyre*)
+		CC_SHARED_OPTS='-fPIC'
+		MK_SHARED_LIB='${CC} -shared -o $@'
+		;;
 	(mingw*)
 		cf_cv_shlib_version=mingw
 		cf_cv_shlib_version_infix=mingw
-- 
2.34.1

